### DISCUSSION TITLE
Don't know how to know if I have got benefits

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [community/community#64911] [GitHub Education] [answered]
### Hi everyone,

Guys I have got approved in GitHub education and I can see this 
![IMG_20230825_084935](https://github.com/community/community/assets/102732282/13aa08c3-9b5c-40e1-a5bc-1ba24a3f8e54)
But I not got any mail from GitHub or I don't see i have got benefits, it's around been 1 day i have got approved, how much I need to wait more?

**Comment by advayc** [comment, marked as answer]
You'll get your benefits about 72 hours after your application. After then you'll get a email along with the PRO badge on your github profile.

**Reply by NaviTheCoderboi** [reply]
Thanks for help

**Reply by Rashish423** [reply]
But It has been more than a month in my case

**Reply by NaviTheCoderboi** [reply]
why do digital ocean ask me for my payment method even i have github education, as a student i dont card

**Comment by nirav-gajera** [comment]


**Reply by NaviTheCoderboi** [reply]
I'll contact them , thanks for help
